game-core/src/main/java/games/strategy/engine/framework/GameRunner.java

```diff
@@ -81,30 +81,27 @@ private GameRunner() {}
    */
   public static void start() {
     SwingUtilities.invokeLater(() -> {
-      final JFrame frame = JFrameBuilder.builder()
-          .title("TripleA")
-          .build();
-      setupPanelModel = new SetupPanelModel(gameSelectorModel, frame);
-      mainFrame = newMainFrame(frame);
+      newMainFrame();
+      setupPanelModel = new SetupPanelModel(gameSelectorModel, mainFrame);
+      mainFrame.add(new MainPanelBuilder().buildMainPanel(setupPanelModel, gameSelectorModel));
+      mainFrame.pack();
       setupPanelModel.showSelectType();
       new Thread(GameRunner::showMainFrame).start();
     });
 
     UpdateChecks.launch();
   }
 
-  private static JFrame newMainFrame(final JFrame frame) {
-    LookAndFeelSwingFrameListener.register(frame);
+  public static void newMainFrame() {
+    mainFrame = JFrameBuilder.builder()
+        .title("TripleA")
+        .build();
+    LookAndFeelSwingFrameListener.register(mainFrame);
 
-    frame.add(new MainPanelBuilder().buildMainPanel(setupPanelModel, gameSelectorModel));
-    frame.pack();
+    mainFrame.setDefaultCloseOperation(WindowConstants.EXIT_ON_CLOSE);
+    mainFrame.setLocationRelativeTo(null);
```

Comment:
This is by default already, this line should not be needed. See `JFrameBuilder#build`